.eslintignore

```diff
@@ -0,0 +1,2 @@
+js/provider/taihe.js
```

Comment:
why excludes files from eslint

Reply:
https://github.com/mikelxk/listen1_chrome_extension/blob/6feb5f32757abf7fc685aadd579211d9c230ced1/js/provider/taihe.js#L30-L32
I used a static class property which isn't quite supported by eslint and I think it's really a burden to add a babel parser.

Reply:
ok, can we just disable some line instead of ignore whole file?

Reply:
That shows it's a parser error, which doesn't belong to a rule

Reply:
https://github.com/tc39/proposal-static-class-features
The static public field is not official added to ES. It is still on stage 3. It is recommended to avoid using it for now.

Consider using the constructor function to create an instance.